purchases/src/main/kotlin/com/revenuecat/purchases/CustomerInfo.kt

```diff
@@ -53,6 +53,39 @@ data class CustomerInfo(
     private val jsonObject: JSONObject,
 ) : Parcelable, RawDataContainer<JSONObject> {
 
+    @IgnoredOnParcel
+    @ExperimentalPreviewRevenueCatPurchasesAPI
+    var virtualCurrencies: Map<String, VirtualCurrencyInfo> = emptyMap()
+        private set
+
+    @OptIn(ExperimentalPreviewRevenueCatPurchasesAPI::class)
+    constructor(
+        entitlements: EntitlementInfos,
+        allExpirationDatesByProduct: Map<String, Date?>,
+        allPurchaseDatesByProduct: Map<String, Date?>,
+        requestDate: Date,
+        schemaVersion: Int,
+        firstSeen: Date,
+        originalAppUserId: String,
+        managementURL: Uri?,
+        originalPurchaseDate: Date?,
+        virtualCurrencies: Map<String, VirtualCurrencyInfo>,
```

Comment:
We place `virtualCurrencies` in its own new @OptIn constructor to preserve the existing constructor. We can move it to the primary constructor and deprecate the existing one once virtual currencies are no longer opt-in